app/src/main/java/org/xbmc/kore/service/library/SyncMovies.java

```diff
@@ -163,7 +163,6 @@ public void onSuccess(List<VideoType.DetailsMovie> result) {
                     // As we're going to recurse, these result objects can add up, so
                     // let's help the GC and indicate that we don't need this memory
                     // (hopefully this works)
-                    result = null;
```

Comment:
As long as the method is recursing it will be in scope. As it is used to sync items over a network recursing may take a while. I believe it was originally added to allow the GC to cleanup already handled results while the method is still recursing.

BTW: remove the comment as well, as it doesn't make sense anymore.